solutions/pipelines-review/pipeline.yaml

```
@@ -26,16 +26,22 @@ spec:
   tasks:
     - name: clone-repository
       taskRef:
-        name: git-clone
-        kind: ClusterTask
+        resolver: cluster
+        params:
+          - name: kind
+            value: task
+          - name: name
+            value: git-clone
+          - name: namespace
+            value: openshift-pipelines
       params:
-        - name: url
+        - name: URL
           value: $(params.GIT_REPO)
-        - name: revision
+        - name: REVISION
           value: $(params.GIT_REVISION)
-        - name: deleteExisting
+        - name: DELETE_EXISTING
           value: 'true'
```

Changing only to the `diff` command between this file and the student file does not show any changes

```suggestion
          value: "true"
```